internal/pubsub/gcppubsub/gcppubsub.go

```diff
@@ -20,35 +20,67 @@ import (
 	"context"
 	"fmt"
 
-	rawgcppubsub "cloud.google.com/go/pubsub/apiv1"
+	raw2 "cloud.google.com/go/pubsub"
```

Comment:
Why do you want to use this client? It's too high-level.

Reply:
It has methods to check the existence of topics and subscriptions, but maybe this is a case of a little redundancy being better than a little dependency. Removed the import and added some redundant code.